.github/workflows/tx-push.yml

```diff
@@ -0,0 +1,25 @@
+name: Get text and upload to transifex
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
We could execute this job only if there are changes in .md files.

Reply:
do you know how to do it? I can look it up if needed, but figured it might be faster if you already know. We should also take into account the notebooks, .py files and the requirements file.

Reply:
This should work:
```yaml
on:
  push:
    branches:
      - main
    paths:
      - '**.md'
      - '**.py'
      - '**.ipynb'
      - requirements-docs.txt
```